app/models/concerns/user/devise_has_domain.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Ce concern nous permet de définir le domaine à utiliser pour un⋅e usager⋅e dans les mailers Devise
+module User::DeviseHasDomain
```

Comment:
Pour montrer qu'on fait une extension d'un module devise existant, on pourrait même utiliser ce nom : 
```suggestion
module User::DeviseInvitableWithDomain
```